### PR TITLE
Navbar

### LINKED ISSUE #130: Navbar
Design inspiration: https://cursor.sh

### PR DESCRIPTION
**Pull request by Nafisay**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,11 +3,13 @@ import FAQ from './pages/FAQ';
 import Home from './pages/Home';
 import Projects from './pages/Projects';
 import Testimonials from './pages/Testimonials';
+import Navbar from './components/Navbar'
```
Comment (harshkhandeparkar) [inline src/App.tsx]:
```suggestion
import Navbar from './components/Navbar';
```

for consistency

File: src/App.tsx [hunk 2]
```diff
@@ -3,11 +3,13 @@ import FAQ from './pages/FAQ';
...
 
 function App() {
   return (
-    <div className="App">
+    <div className="App">
```
Comment (harshkhandeparkar) [inline src/App.tsx]:
```suggestion
    <div className="App">
```

File: src/App.tsx [hunk 3]
```diff
@@ -3,11 +3,13 @@ import FAQ from './pages/FAQ';
...
 import Projects from './pages/Projects';
 import Testimonials from './pages/Testimonials';
+import Navbar from './components/Navbar'
 
 function App() {
   return (
-    <div className="App">
+    <div className="App">      
       <BrowserRouter>
+      <Navbar />
```
Comment (harshkhandeparkar) [inline src/App.tsx]: Indent this and everything inside `BrowserRouter`

File: src/assets/image.png
Comment (harshkhandeparkar) [inline src/assets/image.png:1]: Please remove this file since it is not used

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by rajivharlalka** [comment]
Update? @Nafisay

File: src/assets/login.svg
Comment (harshkhandeparkar) [inline src/assets/login.svg:1]: Please use existing icon libraries instead of using custom icons.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Review by harshkhandeparkar — APPROVED** [review]

**Pull request merged**